application/console/controllers/CronController.php

```diff
@@ -129,4 +129,18 @@ public function actionSendPasswordExpiryEmails()
         $emailer->sendPasswordExpiringEmails();
         $emailer->sendPasswordExpiredEmails();
     }
+
+    public function actionDeleteInactiveUsers()
+    {
+        $logMessage = [ 'action' => 'delete inactive users' ];
+
+        if (\Yii::$app->params['inactiveUserDeletionEnable']) {
```

Comment:
Would this conditional logic be better placed within the `deleteInactiveUsers()` method? That would prevent us from deleting inactive users whenever the config is set to false (which is more what people may expect when setting that configuration value), rather than just preventing the cron job from doing it.

That would also gather the logging all into the same place, rather than logging the start here in the cron controller and logging the end over in the User class.

Reply:
Good. This also allows for cleaner logic (i.e. removing an `if`).

Reply:
Good call. Also allows for cleaner logic (i.e. removing an `if`).